examples/app-dir-mdx/tsconfig.json

```diff
@@ -20,6 +20,12 @@
       }
     ]
   },
```

Comment:
should all of these `tsconfig.json` files also have the `mdx.checkMdx` setting to enable type checking in `.mdx` files in the editor?

```json
  "mdx": {
    // Enable strict type checking in MDX files.
    "checkMdx": true
  }
```